srcpkgs/spdx-licenses-list/template

```diff
@@ -3,14 +3,14 @@
 # run common/scripts/check-custom-licenses after updating to find
```

Comment:
was this run?

Reply:
Sorry for the delay.
I just run the script and added a commit to change the lsof license from custom. Thanks!